app/controllers/api/v1/statuses_controller.rb

```diff
@@ -4,7 +4,7 @@ class Api::V1::StatusesController < ApiController
   before_action -> { doorkeeper_authorize! :read }, except: [:create, :destroy, :reblog, :unreblog, :favourite, :unfavourite]
   before_action -> { doorkeeper_authorize! :write }, only:  [:create, :destroy, :reblog, :unreblog, :favourite, :unfavourite]
   before_action :require_user!, except: [:show, :context, :card, :reblogged_by, :favourited_by]
-  before_action :set_status, only:      [:show, :context, :card, :reblogged_by, :favourited_by]
+  before_action :set_status, only:      [:show, :update, :context, :card, :reblogged_by, :favourited_by]
```

Comment:
Oops, that's a mistake brought in by something else I was testing. I'll remove it. P.S: If you're against status editing, you might want to comment as such on #742!